// @cosmjs/cosmwasm-stargate@0.36.1/build/modules/wasm/messages.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.wasmTypes = void 0;
exports.isMsgStoreCodeEncodeObject = isMsgStoreCodeEncodeObject;
exports.isMsgInstantiateContractEncodeObject = isMsgInstantiateContractEncodeObject;
exports.isMsgInstantiateContract2EncodeObject = isMsgInstantiateContract2EncodeObject;
exports.isMsgUpdateAdminEncodeObject = isMsgUpdateAdminEncodeObject;
exports.isMsgClearAdminEncodeObject = isMsgClearAdminEncodeObject;
exports.isMsgMigrateEncodeObject = isMsgMigrateEncodeObject;
exports.isMsgExecuteEncodeObject = isMsgExecuteEncodeObject;
const tx_1 = require("cosmjs-types/cosmwasm/wasm/v1/tx");
exports.wasmTypes = [
    ["/cosmwasm.wasm.v1.MsgClearAdmin", tx_1.MsgClearAdmin],
    ["/cosmwasm.wasm.v1.MsgExecuteContract", tx_1.MsgExecuteContract],
    ["/cosmwasm.wasm.v1.MsgMigrateContract", tx_1.MsgMigrateContract],
    ["/cosmwasm.wasm.v1.MsgStoreCode", tx_1.MsgStoreCode],
    ["/cosmwasm.wasm.v1.MsgInstantiateContract", tx_1.MsgInstantiateContract],
    ["/cosmwasm.wasm.v1.MsgInstantiateContract2", tx_1.MsgInstantiateContract2],
    ["/cosmwasm.wasm.v1.MsgUpdateAdmin", tx_1.MsgUpdateAdmin],
];
function isMsgStoreCodeEncodeObject(object) {
    return object.typeUrl === "/cosmwasm.wasm.v1.MsgStoreCode";
}
function isMsgInstantiateContractEncodeObject(object) {
    return (object.typeUrl === "/cosmwasm.wasm.v1.MsgInstantiateContract");
}
function isMsgInstantiateContract2EncodeObject(object) {
    return (object.typeUrl === "/cosmwasm.wasm.v1.MsgInstantiateContract2");
}
function isMsgUpdateAdminEncodeObject(object) {
    return object.typeUrl === "/cosmwasm.wasm.v1.MsgUpdateAdmin";
}
function isMsgClearAdminEncodeObject(object) {
    return object.typeUrl === "/cosmwasm.wasm.v1.MsgClearAdmin";
}
function isMsgMigrateEncodeObject(object) {
    return object.typeUrl === "/cosmwasm.wasm.v1.MsgMigrateContract";
}
function isMsgExecuteEncodeObject(object) {
    return object.typeUrl === "/cosmwasm.wasm.v1.MsgExecuteContract";
}
